src/ConstructiveSolidGeometry/IO.jl

```diff
@@ -17,7 +17,11 @@ const CSG_dict = Dict{String, Any}(
 
 function get_geometry_key(::Type{T}, dict::AbstractDict, input_units::NamedTuple) where {T}
     g = collect(filter(k -> k in keys(CSG_dict), keys(dict)))
-    @assert !(length(g) > 1) "Too many geometry entries in dictionary: $(length(g))."
+    filter!(k -> !(
+        (k == "translate" && !any(broadcast(key -> key in keys(CSG_dict), keys(dict["translate"])))) ||
+        (k == "rotate" && !any(broadcast(key -> key in keys(CSG_dict), keys(dict["rotate"]))))
+    )  , g) # exclude translate and rotate that have no primitives inside
+    @assert !(length(g) > 1) "Too many ($(length(g))) geometry entries in dictionary: $(g)."
```

Comment:
This might need to be adjusted to changes in #168.

Reply:
I did not put it into #168 to avoid merge conflicts.

Reply:
Still, you added "translation", "origin", etc.
We have to see that we get this straight one after another.

Reply:
Yes, but only inside primitives. But lets see.